presets/inference/text-generation/requirements.txt

```diff
@@ -14,7 +14,6 @@ numpy==1.22.4
 bitsandbytes==0.42.0
 
 # Less critical, can be latest
-deepspeed
```

Comment:
Including deepspeed package causes breaking version with transformers - to be included in future release